foodnet/membership/forms.py

```diff
@@ -0,0 +1,87 @@
+from django import forms
+from .models import UserProfile
+from django.core.mail import send_mail
+from allauth.account.forms import BaseSignupForm
+from allauth.account.adapter import get_adapter
+from allauth.account.utils import user_email, setup_user_email, \
+    cleanup_email_addresses
+from allauth.account.models import EmailAddress
+from captcha.fields import ReCaptchaField
+
+from .models import MemberCategory, Division, UserProfile
+
+
+class ProfileForm(forms.Form):
```

Comment:
So it was originally but because we still trying to work out how the profile should look/behave I was thinking of adding more fields and validations so I thought it will be easier to do it as classic Form.